### DISCUSSION TITLE
8주차 과제

### DISCUSSION THREAD

**Discussion by youngsangcho** [youngsangcho/generativedesign2_2021#26] [Show and tell]
#과제
Dynamic Typography 코드
+ A-Z 까지 타이핑 되는 코드 작성
+ 인터랙션 / 애니메이션 / 랜덤화 시킬 아이디어 글로 작성 (필요시 스케치) 
+ 블로그 글에 p5js 에디터 링크 포함

11월 15일 밤 12시까지.
더 일찍 제출하고 많이 질문할 수록 많은 도움을 받을 수 있음!

**Comment by SeyounJung01** [comment]
정세윤 https://blog.naver.com/seyounjj2001/222568363842

**Comment by KimHyunJuhg** [comment]
김현정
https://blog.naver.com/khj317888/222568903513
내일 수업 때까지 일단 계속 수정해 가도록 하겠습니다!

**Comment by eunjinhong** [comment]
홍은진
https://blog.naver.com/tnekf1012/222569123555

**Comment by imyeeunok** [comment]
김예은
https://blog.naver.com/010312kye/222569257423

수업 전까지 더 발전시켜서 올리겠습니다 !

**Comment by Joeun410** [comment]
김조은
https://blog.naver.com/PostView.naver?blogId=joanne3456&Redirect=View&logNo=222569295310&categoryNo=17&isAfterWrite=true&isMrblogPost=false&isHappyBeanLeverage=true&contentLength=16200

**Comment by jwejwepark** [comment]
박주혜
https://blog.naver.com/glqkfl0726/222569303059
